infra/docker-compose/docker-compose.batch.yml

```diff
@@ -0,0 +1,25 @@
+version: "3.7"
+
+services:
+  batch-serving:
+    image: ${FEAST_SERVING_IMAGE}:${FEAST_VERSION}
+    volumes:
+      - ./serving/${FEAST_BATCH_SERVING_CONFIG}:/etc/feast/application.yml
+      - ./serving/${FEAST_BATCH_STORE_CONFIG}:/etc/feast/store.yml
```

Comment:
I think this is not necessary anymore after the config refactor

Reply:
We don't run `master` on Docker Compose, so the configuration refactor shouldn't affect things here.

Reply:
ooh okay. got it

Reply:
Thanks for taking a look though :)

Going to resolve these...